Vsix/CodeConverterPackage.cs

```diff
@@ -46,12 +46,6 @@ namespace ICSharpCode.CodeConverter.VsExtension
     [ProvideUIContextRule(VbEditorMenuVisibilityGuid, name: nameof(VbEditorMenuVisibilityGuid),
         expression: "Vb", termNames: new[] { "Vb" },
         termValues: new[] { "ActiveEditorContentType:Basic" })]
-    [ProvideUIContextRule(CsFileMenuVisibilityGuid, name: nameof(CsFileMenuVisibilityGuid),
-        expression: "DotCs", termNames: new[] { "DotCs" },
-        termValues: new[] { "HierSingleSelectionName:.cs$"})]
-    [ProvideUIContextRule(VbFileMenuVisibilityGuid, name: nameof(VbFileMenuVisibilityGuid),
-        expression: "DotVb", termNames: new[] { "DotVb" },
-        termValues: new[] { "HierSingleSelectionName:.vb$"})]
```

Comment:
Could we restore these (and the vsct parts) and just use `ActiveProjectCapability:CSharp` and `ActiveProjectCapability:VB` to keep them only visible for the appropriate project type? Same goes for the folder node one I think.

Reply:
Good idea. I think that should work just fine with multi-select.
I will take a look.